stdlib/universe/join_use_previous_test.flux

```diff
@@ -0,0 +1,76 @@
+package universe_test
+
+import "csv"
+import "testing"
+
+option now = () => (2030-01-01T00:00:00Z)
```

Comment:
Can you set the now value to when the data is supposed to end instead of to 2030?

Reply:
Yeah, totally. I pulled this from another test, so I'm not all that attached to it. I'm curious, though: is this a sanitation thing, or a performance thing? What's the reasoning here? I would _assume_ that, because there is a finite number of data points, it's really not a performance hit. If that's not the case, I'd be interested to know why.